### PR TITLE
Azure docs

### PR DESCRIPTION
**Pull request by croomes**
First draft of Azure doc, feedback welcome.  Still a bit more testing to do but this will at least get us started.
Make sure if you're testing to use the `feature/devicedir_and_roles` branch of https://github.com/storageos/helm-chart

\cc @stuartleeks

### CODE REVIEW

File: _docs/install/cloud/azure.md [hunk 1]
```diff
@@ -5,4 +5,204 @@ anchor: install
...
+
+## acs-engine
+
+`acs-engine` is an open-source tool from Microsoft that uses to deploy AKS clusters, and it can be used to deploy your own Kubernetes cluster on Azure.  It allows
+you to specify the version of Kubernetes and specific features.
+
+For more information and installation instructions, consult the
+[acs-engine kubernetes guide](https://github.com/Azure/acs-engine/blob/master/docs/kubernetes/deploy.md)
+
+There are various [examples of kubernetes cluster definitions](https://github.com/Azure/acs-engine/tree/master/examples) for acs-engine, but the important thing is to specify kubernetes 1.10 as StorageOS requires this version to support running with kubelet in a container.
```
Comment (oicheryl) [inline _docs/install/cloud/azure.md]: Capitalize Kubernetes (and line above).

File: _docs/install/cloud/azure.md [hunk 2]
```diff
@@ -5,4 +5,204 @@ anchor: install
...
+    "servicePrincipalProfile": {
+      "clientId": "",
+      "secret": ""
+    }
+  }
+}
+```
+
+To deploy a cluster through acs-engine you need your Azure Subscription ID. 
+The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) is a great way to get the ID of your subscription. If you don't have the Azure CLI installed then you can sign in to [Azure Cloud Shell](https://shell.azure.com) to run the commands there:
```
Comment (oicheryl) [inline _docs/install/cloud/azure.md]: "The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) is a great way to get the ID of your subscription."
=>
"You can retrieve your subscription ID through the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)."

File: _docs/install/cloud/azure.md [hunk 3]
```diff
@@ -5,4 +5,204 @@ anchor: install
...
+    --auto-suffix --api-model kubernetes.json
+```
+
+The deployment also creates a KUBECONFIG for your cluster under the `_output` directory. The exact filename depends on the dns prefix and location you specified. To set the KUBECONFIG for the example above:
+
+```bash
+KUBECONFIG=~/mydir/_output/storageos/kubeconfig/kubeconfig.westus2.json
+```
+
+Running `kubectl cluster-version` should now connect to your new cluster!
```
Comment (oicheryl) [inline _docs/install/cloud/azure.md]: Replace ! with . (period)

File: _docs/install/cloud/azure.md [hunk 4]
```diff
@@ -5,4 +5,204 @@ anchor: install
...
+install StorageOS.
+
+Once Helm is installed and configured for the cluster, checkout the StorageOS
+Helm Chart and install:
+
+```bash
+git clone https://github.com/storageos/helm-chart.git storageos
+cd storageos
+helm install . --name storageos-test \
+  --set image.repository=soegarots/node \
```
Comment (oicheryl) [inline _docs/install/cloud/azure.md]: Replace soegarots with storageos

File: _docs/install/cloud/azure.md [hunk 5]
```diff
@@ -5,4 +5,204 @@ anchor: install
...
+  --set image.tag=ad3f0ccb6 \
```
Comment (stuartleeks) [inline _docs/install/cloud/azure.md]: @oicheryl /@croomes - should we be specifying a specific tag here?

Comment (oicheryl) [reply _docs/install/cloud/azure.md]: Good catch - 0.10.0 should be the tag.

**Comment by stuartleeks** [comment]
How is this looking with the latest couple of commits that have been merged?

**Comment by oicheryl** [comment]
Looks great, thanks for all the work @stuartleeks @croomes!

To get install/cloud/azure.md to show up in the left hand column you need to add it to this file:

https://github.com/storageos/storageos.github.io/blob/master/_guides/002-install_and_run.md

**Comment by croomes** [comment]
Thanks @stuartleeks !

**Pull request merged**